core/controller/build.gradle

```diff
@@ -43,6 +43,7 @@ dependencies {
     compile 'com.lightbend.akka.discovery:akka-discovery-kubernetes-api_2.12:0.11.0'
     compile 'com.lightbend.akka.discovery:akka-discovery-marathon-api_2.12:0.11.0'
     compile project(':common:scala')
+    compile project(':core:invoker')
```

Comment:
should this be done only when necessary? at least should have a comment.

Reply:
LeanBalancer has dependency on InvokerReactive, it can't be optional dependency